ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicationManager.java

```diff
@@ -236,4 +240,18 @@ public long getRemoteReplicaLagFromLocalInBytes(PartitionId partitionId, String
     // TODO get replica lag from cosmos?
     return -1;
   }
+
+  /**
+   * Check if replication is allowed from given datacenter.
+   * @param datacenterName datacenter name to check.
+   * @return true if replication is allowed. false otherwise.
+   */
+  private boolean shouldReplicateFromDc(String datacenterName) {
+    if (cloudConfig.cloudIsVcr) {
```

Comment:
cloudIsVcr will be true in both cases.  Perhaps it's time to add another config property like isBackup?
But I think we can use the new logic for both scenarios, and update the DR config to tell it which DC to use.

Reply:
sure.. as of now I don't see any other explicit need for a separate config for backup, so I will use the same config for all cases. We can revisit if we need a separate config for backup if an explicit need arises.